tests/DependencyInjection/EightPointsGuzzleExtensionTest.php

```diff
@@ -34,6 +34,9 @@ public function testGuzzleExtension()
         $this->assertInstanceOf(Client::class, $testApi);
         $this->assertEquals(new Uri('//api.domain.tld/path'), $testApi->getConfig('base_uri'));
 
+        $this->assertTrue($container->hasAlias(ClientInterface::class.' $testApiClient'));
```

Comment:
Tests are failing for SF 4.1, because in SF 4.1 alias is not created.

Reply:
OK, I'm doing stupid things :) Does a best practice exist for excluding specific Symfony versions for a test?